Samples/TrendingMovies/TrendingMovies/ImageProcessing/ImageEffects.swift

```diff
@@ -6,13 +6,13 @@ struct ImageEffects {
                                       blurRadius: CGFloat,
                                       tintColor: UIColor? = nil,
                                       saturationDeltaFactor: CGFloat = 1.0) -> UIImage? {
-        let span = Tracer.startSpan(name: "create-blurred-backdrop")
-        span.annotate(key: "width", value: String(Double(image.size.width)))
-        span.annotate(key: "height", value: String(Double(image.size.height)))
-        span.annotate(key: "blurRadius", value: String(Double(blurRadius)))
-        span.annotate(key: "downsamplingFactor", value: String(Double(downsamplingFactor)))
-        span.annotate(key: "saturationDeltaFactor", value: String(Double(saturationDeltaFactor)))
-        defer { span.end() }
+//        let span = Tracer.startSpan(name: "create-blurred-backdrop")
+//        span.annotate(key: "width", value: String(Double(image.size.width)))
+//        span.annotate(key: "height", value: String(Double(image.size.height)))
+//        span.annotate(key: "blurRadius", value: String(Double(blurRadius)))
+//        span.annotate(key: "downsamplingFactor", value: String(Double(downsamplingFactor)))
+//        span.annotate(key: "saturationDeltaFactor", value: String(Double(saturationDeltaFactor)))
+//        defer { span.end() }
```

Comment:
`l`: Don't we want to delete this code?

Reply:
Maybe I'll also make these dependent on the testing launch arg.